documentation-site/components/yard/config/form-control.ts

```diff
@@ -0,0 +1,69 @@
+import {FormControl} from 'baseui/form-control';
+import {Input} from 'baseui/input';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const TextareaConfig: TConfig = {
+  imports: {
+    'baseui/form-control': {named: ['FormControl']},
+  },
+  scope: {
+    FormControl,
+    Input,
+  },
+  theme: [],
+  props: {
+    children: {
+      value: '<Input />',
+      type: PropTypes.ReactNode,
+      description: 'Content wrapped by FormControl.',
+      imports: {'baseui/input': {named: ['Input']}},
+    },
+    label: {
+      value: 'label',
+      type: PropTypes.String,
+      description: 'A label rendered above the input field.',
+    },
+    caption: {
+      value: 'caption',
+      type: PropTypes.String,
+      description: 'A caption rendered below the input field.',
+    },
+    positive: {
+      value: undefined,
+      type: PropTypes.String,
```

Comment:
Same for the `positive` and `error`. But apparently it can also be a `boolean` based on the code, but it's not reflected in types.